### ISSUE TITLE
Is DWM1000 ok?

### ISSUE THREAD

**Issue by marsprobe** [vortex314/ea_actor#1]
Hi Lieven, we are including dwm1000 module on esp8266. Now, we found the dwm1000 module can't send packets. I found this repo and the words: trying and trying ,no reply received. Did you solve the question?

**Comment by vortex314** [comment]
I was able to send messages from "tag"  DMW1000-ESP8266 to "anchor" DMW1000-ESP8266 , but the reply to this poll-request by "anchor" times out. To what I understand the anchor has 150 micro-seconds to respond, this to enable distance measurement. 

Since the ESP8266 is a very badly documented chip, I couldn't upgrade the  speed enough reliably or use DMA ( totally undocumented ), so I abandoned and switch to STM32 now.
